### PR TITLE
Add an intro to all crates

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
cc @sophie-h

### CODE REVIEW

File: gdk4-wayland/src/lib.rs [hunk 1]
```diff
@@ -1,5 +1,11 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
+// rustdoc-stripper-ignore-next
+//! # Rust GDK 4 Wayland bindings
+//!
+//! This library contains safe Rust bindings for [GDK 4 Wayland](https://docs.gtk.org/gdk4-wayland/) which
+//! is the Wayland GDK backend specific functions.
```
Comment (Hofer-Julian) [inline gdk4-wayland/src/lib.rs:7]:
```suggestion
//! This library contains safe Rust bindings for [GDK 4 Wayland](https://docs.gtk.org/gdk4-wayland/).
//! GDK is an intermediate layer that isolates GTK from the details of the windowing system.
//! GDK Wayland contains functions specific to the Wayland backend.
```

File: gdk4-x11/src/lib.rs [hunk 2]
```diff
@@ -1,5 +1,11 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
+// rustdoc-stripper-ignore-next
+//! # Rust GDK 4 X11 bindings
+//!
+//! This library contains safe Rust bindings for [GDK 4 X11](https://docs.gtk.org/gdk4-x11/) which
+//! is the X11 GDK backend specific functions.
```
Comment (Hofer-Julian) [inline gdk4-x11/src/lib.rs:7]:
```suggestion
//! This library contains safe Rust bindings for [GDK 4 X11](https://docs.gtk.org/gdk4-x11/).
//! GDK is an intermediate layer that isolates GTK from the details of the windowing system.
//! GDK X11 contains functions specific to the X11 backend.
```

File: gdk4/src/lib.rs [hunk 3]
```diff
@@ -1,5 +1,11 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
+// rustdoc-stripper-ignore-next
+//! # Rust GDK 4 bindings
+//!
+//! This library contains safe Rust bindings for [GDK 4](https://docs.gtk.org/gdk4/) which
+//! is an intermediate layer that isolates GTK from the details of the windowing system.
```
Comment (Hofer-Julian) [inline gdk4/src/lib.rs:7]:
```suggestion
//! This library contains safe Rust bindings for [GDK 4](https://docs.gtk.org/gdk4/).
//! GDK is an intermediate layer that isolates GTK from the details of the windowing system.
```

File: gsk4/src/lib.rs [hunk 4]
```diff
@@ -1,5 +1,11 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
+// rustdoc-stripper-ignore-next
+//! # Rust GSK 4 bindings
+//!
+//! This library contains safe Rust bindings for [GSK4](https://docs.gtk.org/gsk4/) which
+//! is an intermediate layer that isolates GTK from the details of the OpenGL or Vulkan implementation.
```
Comment (Hofer-Julian) [inline gsk4/src/lib.rs:7]:
```suggestion
//! This library contains safe Rust bindings for [GSK4](https://docs.gtk.org/gsk4/).
//! GSK is an intermediate layer that isolates GTK from the details of the OpenGL or Vulkan implementation.
```

File: gdk4-wayland/src/lib.rs (hunk 1 above)
Comment (bilelmoussaoui) [reply gdk4-wayland/src/lib.rs:7]: Any reason for duplicating the GDK definition here?

Comment (Hofer-Julian) [reply gdk4-wayland/src/lib.rs:7]: It is a separate file after all.

**Review by GuillaumeGomez — APPROVED** [review]

**Pull request merged**